src/main/java/se/sundsvall/supportmanagement/api/model/communication/conversation/ConversationType.java

```diff
@@ -0,0 +1,10 @@
+package se.sundsvall.supportmanagement.api.model.communication.conversation;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+@Schema(description = "ConversationType model", enumAsRef = true)
+public enum ConversationType {
+
+	D2D,	// Dragon to Dragon
+	D2C		// Dragon to citizen
```

Comment:
Will citizen not be able to communicate with dragons? Or will they be able to respond but not start a conversation?

Reply:
Changed to INTERNAL, EXTERNAL